README.md

```diff
@@ -151,6 +151,10 @@ deploying your application:
 cf create-service cyberark-conjur community conjur
 ```
 
+Note that service instances can not currently be shared between spaces so a
```

Comment:
I would remove "currently" from this note. It makes it sound like a temporary decision that we intend to change.